### DISCUSSION TITLE
[feature request] Add details about the uploading task

### DISCUSSION THREAD

**Discussion by vendornet** [filebrowser/filebrowser#1574] [Ideas]
I’ve been using filebrowser for a week and it’s a great tool to manage my uploads. I use it to upload large files to my server. I’ve tested few alternatives and filebrowser shines with its simplicity and reliability. I also like the straightforward docker installation.

It would be really great when uploading a file to see stats:
- Current % of the uploaded file
- Current upload speed - like 2.5 mb/s
- Current size uploaded and total - like 500 mb of 1500 mb uploaded
- Estimated time to complete - like 25 min. remaining

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
It would be really great when uploading a file to see stats:
- Current % of the uploaded file
- Current upload speed - like 2.5 mb/s
- Current size uploaded and total - like 500 mb of 1500 mb uploaded
- Estimated time to complete - like 25 min. remaining

**Describe alternatives you've considered**
Currently I can only look at the blue bar at the top, but it is really high level overview.

**Additional context**
Let me know if you need more info

**Comment by devtechk** [comment]
Hey! same here! Will be really usefull add this stats!
